core/src/lib/bareos_resource.h

```diff
@@ -3,7 +3,7 @@
 
    Copyright (C) 2000-2010 Free Software Foundation Europe e.V.
    Copyright (C) 2011-2012 Planets Communications B.V.
-   Copyright (C) 2013-2023 Bareos GmbH & Co. KG
+   Copyright (C) 2013-2024 Bareos GmbH & Co. KG
```

Comment:
This needs to be reverted.

Reply:
that's going to happen when the fixups are squashed and bareos-check-sources is re-applied